spring-boot-security-taglibs/src/main/webapp/WEB-INF/views/home.jsp

```diff
@@ -0,0 +1,33 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8"
+    pageEncoding="UTF-8"%>
+<%@ taglib prefix="sec"
+    uri="http://www.springframework.org/security/tags"%>
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
+<sec:csrfMetaTags />
+<title>Home Page</title>
+</head>
+<body>
+    <sec:authorize access="isAnonymous()">
+        ANONYMOUS Content
+    </sec:authorize>
+    <sec:authorize access="isAuthenticated()">
+        AUTHENTICATED Content 
+        <sec:authorize access="hasRole('ADMIN')">
+            Content for users who have the "ADMIN" role.
+        </sec:authorize>
+        <h2>
+            Welcome back, <sec:authentication property="name" />
+        </h2>
+        <form>
+            <sec:csrfInput />
```

Comment:
What I mean here is that without an endpoint that actually requires CSRF protection, then they can't really see this one in action. So, what thinking, what if you did this:

```html
<form method="post">
  <sec:csrfInput />
  <input type="submit" value="Do a post">
</form>
```

Then, folks can interact with the form.

Let me know if I'm misunderstanding or not being clear.